api/app/src/app.ts

```diff
@@ -32,8 +32,9 @@ mountRoutes(app);
 module.exports = app;
 
 // route used for health checks
-app.get("/", (_req: Request, res: Response) => {
-  res.status(200).send("OK");
+app.get("/", (req: Request, res: Response) => {
+  if (req.accepts("application/json")) return res.json({ status: "OK" });
+  return res.status(200).send("OK");
```

Comment:
Simple, yet important change. I don't foresee issues, but can rollback if needed.

Reply:
This is not required for this PR AFAIR, midway through the coding I was doing some testing and wanted the response to be JSON - but it wasn't.

We talked about having the responses of our API being JSON, so I thought it was a good opportunity to fix it.

If you foresee an issue I can def rollback and ticket it; but if its just bc of scope, I suggest we fix sooner than later.

Reply:
Yeah this shouldn't be an issue, was just wondering